styles.css

```diff
@@ -0,0 +1,158 @@
+html {
+	font-family: "Trebuchet MS", sans-sarif;
+	box-sizing: border-box;
+	font-size: 10px;
+}
+
+body {
+	background-color:#0a415c;
+	margin: 0;
+	padding: 0;
+	color:#cfd2d4;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
+p {
+	font-size: 1.5rem;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;	
+	align-items: center;
+}
+
+.nav-bar {
+	width: 20%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	gap: 20px;
+	font-size: 2rem;
+	padding: 20px;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+	color: #cfd2d4;
+}
+.nav-bar a:hover {
+	color:#048230;
+}
+
+
+.cv-download-button {
+	width: 150px;
+	height: 30px;
+	border: 2px solid #048230;
+	background-color: #0a415c;
+	color: #048230;
+	font-size: 1.6rem;
+	cursor: pointer;
+	border-radius: 20px;
+	margin: 0 20px;
+}
+
+
+.cv-download-button:hover {
+	background-color:#048230;
+	color: #cfd2d4;
+}
+
+header {
+	position: relative;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-bottom: 20px;
+}
+
+.header-title {
+	position: absolute;
+	border: 2px solid;
+	order: 1;
+	width: 500px;
+	height: 60px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-size: 4rem;
+	text-transform: uppercase;
+}
+
+main {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-evenly;
+	max-width: 2000px;
+}
+
+header img {
+	width: 100%;
+}
+
+.portfolio {
+	width: 55%;
+	height: 800px;
+	padding: 20px;
+	overflow: scroll;
+}
+
+.portfolio-card {
+	border: 1px solid;
+	padding: 20px;
+	margin: 20px 0px;
+}
+
+.portfolio-card img{
+	width: 40%;
+}
+
+.about {
+	border: 1px solid;
+	width: 30%;
+	padding: 20px;
+	height: 800px;
+}
+
+.about img {
+	width: 60%;
+}
+
+.contact {
+	background-color: #6a6c6e;
+	margin: 0;
+	padding: 20px 60px;
+}
+
+.social-link-icons {
+	width: 200px;
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-between;
+}
+
+@media screen and (max-width: 600px){
+	html {
+		font-size: 8px;
+	}
+	
+	.portfolio {
+		width: 90vw;
+	}
+	
+	.about {
+		width: 90vw;
+	}
```

Comment:
instead of having a width of 90vw, it would be better to have width 100% (90vw is a highly specific number and 100% looks the same to me)